src/components/ProjectList/listOfProjects.js

```diff
@@ -584,6 +584,34 @@ const projectList = [
     githubLink: 'https://github.com/django/django',
     description: 'A high-level Python Web framework that encourages rapid development and clean, pragmatic design',
     tags: ['python', 'web development']
-  }
+  },
+  {
+    name: 'Plotly',
+    imageSrc: 'https://avatars3.githubusercontent.com/u/5997976?s=200&v=4',
+    githubLink: 'https://github.com/plotly',
+    description: 'A high-level Python declarative charting library',
+    tags: ['python', 'javascript', 'graph', 'graph-based', 'graph visualization']
+  },
+  {
+    name: 'NetworkX',
+    imageSrc: 'https://avatars3.githubusercontent.com/u/388785?s=200&v=4',
+    githubLink: 'https://github.com/networkx/networkx',
```

Comment:
Please add `/contribute` at the end of the URL

Reply:
@Roshanjossey You mean 'Networkx/contribute', 'Igraph/contribute', 'D3/contribute' right?

Reply:
@Roshanjossey Ok, I got it :sweat_smile: Now, I solved. But d3 don't have 'contribute'.

Reply:
@hgcassiopeia, I'm hoping that d3 will add more issue with `good first issues` flag and there will be issues in `/contribute` link in the future.

This is a change make by GitHub recently and we're moving forward with it.